lib/linux.js

```diff
@@ -38,7 +38,17 @@ const trash = async (filePath, trashPaths) => {
 };
 
 export default async function linux(paths) {
-	const mountPointMap = new Map(procfs.processMountinfo().map(info => [info.devId, info.mountPoint]));
+	// • filter irrelevant
+	// • “first one wins” if id comes along twice
```

Comment:
The comment here is not clear. I think both lines could be a single code comment that explains it better.

Reply:
I politely disagree on this one.

It's clearly 2 distinct aspects:
• filter irrelevant drives [line :46 does that, stuff that starts with `/snap`,`/run` etc...
• “first one wins” [the entire new loop :45-:50 does that, on drives that are not clearly „irrelevant“] but come along multiple times. Thus „first one wins“ as in „second one doesn't wipe out first“ (since only one can be considered for finding the right trash path…)

tl;dr: Keep it two lines, but I can make the explanation a bit longer. Just like that.